common/openapi.yml

```diff
@@ -759,6 +759,9 @@ components:
             string
           format: uuid
           example: 00000000-0000-0000-0000-000000000000
+        engine_id:
+          type:
+            string
```

Comment:
I'm not sure I followed why this is necessary here?

If it's just to make the field available for internal logic then I would have thought we should be able to pass the value around without changing the interface?